README.md

```diff
@@ -20,7 +20,7 @@ ftl init kotlin . alice
 
 ### Deploy and test the module
 ```sh
-ftl deploy ftl-module-alice
+ftl deploy alice
```

Comment:
Wanna give these instructions a go and see how they work for you? https://github.com/TBD54566975/ftl/blob/main/CONTRIBUTING.md

If they work well, maybe we can update the README.md at the root to point there instead of having 2 different approaches :)

Reply:
Even those are a bit out of date now that I look at it. Here's a quick set of commands to hopefully get you started and we can use them to update the README(s) as needed. I'm also happy to update these if you'd like as well...

Create a new project to work with ftl

```sh
mkdir myproject && cd myproject
hermit init .
hermit install go && hermit install ftl
ftl init go . mymodule
ftl dev . --recreate
```

These assume you have the shell hooks for hermit setup. If not, you'll need to `. ./bin/activate-hermit` in your project folder

Reply:
> Wanna give these instructions a go and see how they work for you? https://github.com/TBD54566975/ftl/blob/main/CONTRIBUTING.md

Yep, that works for me. 👍 

I like the idea of having some sort of instructions in the README to set things up from scratch. 

I do think we should reference the development prerequisites for setting up the hermit environment (and maybe splitting this out from the `ftl dev ./examples/go`)

The way I see this is, you have your development guide which is focused on people developing on ftl. And then you have the README which can be used for people like me (users) to start a project from scratch without searching through the docs. 

I think something like this would work well... 

```
## Development guide 

> Please make sure you have Hermit setup for this work. 

mkdir myproject && cd myproject

hermit init .

hermit install go && hermit install ftl

ftl init go . mymodule

Talk about some hot reloading

Talk about deploying it locally (if we even need this when we have hot reloading)

```

^ Those are just thoughts, let me sit on it a bit and update the PR of what makes sense in my mind.

Reply:
> And then you have the README which can be used for people like me (users) to start a project from scratch without searching through the docs.

That is what the README does currently do (`ftl` is installed via `brew tap` rather than `hermit`) - it's just that it's bitrotted. I'm working on an adjacent PR that will fix this going forward, I'll tag you on it.

(thanks for trying this out BTW, having newcomers try things out is beneficial)